lib/matplotlib/colorbar.py

```diff
@@ -304,7 +304,47 @@ def tick_values(self, vmin, vmax):
         return ticks
 
 
-class ColorbarBase(cm.ScalarMappable):
+class _ColorbarMappableDummy(object):
+    """
+    Private class to hold deprecated ColorbarBase methods that used to be
+    inhereted from ScalarMappable.
+    """
+    @cbook.deprecated("3.1", alternative="mappable.set_norm")
```

Comment:
For alternative, did you mean `ScalarMappable.set_norm` here, as below?

Reply:
Fixed both points!  Thanks,